### PR TITLE
HTML tables for ANSSI Rules in RHEL7

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Adds HTML table listing all Rules that reference an ANSSI document
  - `table-rhel7-anssirefs.html`
- Adds a profile for each RHEL7 ANSSI Profile, listing their Rules and its ANSSI references
  - `table-rhel7-anssirefs-nt28_minimal.html`
  - `table-rhel7-anssirefs-nt28_intermediary.html`
  - `table-rhel7-anssirefs-nt28_enhanced.html`
  - `table-rhel7-anssirefs-nt28_high.html`

#### Rationale:

 - `table-rhel7-anssirefs.html` is useful to track which SSG Rules covers which ANSSI requirements.
 - Tables by Profiles are useful to track what SSG Rules are covered in each Security Level defined by ANSSI NT28.

EDIT: table names changed from   `table-rhel7-anssirefs-anssi_nt28_minimal.html` to    `table-rhel7-anssirefs-nt28_minimal.html`

### CODE REVIEW

File: rhel7/CMakeLists.txt [hunk 1]
```diff
@@ -14,12 +14,18 @@ ssg_build_html_table_by_ref(${PRODUCT} "nist")
...
 
 ssg_build_html_nistrefs_table(${PRODUCT} "common")
 ssg_build_html_nistrefs_table(${PRODUCT} "ospp-${PRODUCT}")
 ssg_build_html_nistrefs_table(${PRODUCT} "C2S")
 ssg_build_html_nistrefs_table(${PRODUCT} "stig-${PRODUCT}-disa")
 
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_minimal")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_intermediary")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_enhanced")
+ssg_build_html_anssirefs_table(${PRODUCT} "anssi_nt28_high")
```
Comment (mpreisler) [inline rhel7/CMakeLists.txt]: since the anssi_nt28 is prefix is in evrey single one of them, wouldn't it make sense to prepend it inside the macro?

Comment (yuumasato) [reply rhel7/CMakeLists.txt]: We can prepend it inside the macro. But can we assume that ANSSI Profiles for other products will also have 'anssi' prefix?

Comment (dahaic) [reply rhel7/CMakeLists.txt]: I agree with both of you :) `anssi` can be moved into macro, but I would leave `nt28` here. Assumption no other profile will be implemented is too strong to me.

Comment (yuumasato) [reply rhel7/CMakeLists.txt]: I have included `anssi_` inside macro. This should have make name of ANSSI tables less redundant, updated in PR description.

**Comment by dahaic** [comment]
Thanks @yuumasato !

**Comment by dahaic** [comment]
@mpreisler please note there were some references added for reviewers convenience, don't be surprised :)

**Comment by mpreisler** [comment]
> @mpreisler please note there were some references added for reviewers convenience, don't be surprised :)

good stuff, this actually really helps me, I like the style @yuumasato chose for the refs

**Pull request merged**